src/App.tsx

```diff
@@ -1,61 +1,133 @@
 import './App.scss';
+import { useState, useEffect } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { User } from './components/types/User';
+import { Todo } from './components/types/Todo';
+
+import { TodoList } from './components/TodoList/TodoList';
 
 export const App = () => {
+  const [copyTodos, setCopyTodos] = useState([...todosFromServer]);
+  const [selectedUserName, setSelectedUserName] = useState('');
+  const [newTitle, setNewTitle] = useState('');
+  const [errorTitle, setErrorTitle] = useState(false);
+  const [errorUser, setErrorUser] = useState(false);
+
+  const maxId = Math.max(...todosFromServer.map(todo => +todo.id));
+  const findUser = (userName: string): User | null => {
+    return usersFromServer.find(user => user.name === userName) || null;
+  };
+
+  const checkError = () => {
+    if (!newTitle) {
+      setErrorTitle(true);
+    }
+
+    if (!selectedUserName) {
+      setErrorUser(true);
+    }
+  };
+
+  const createTodo = () => {
```

Comment:
There is no need for 'return null' in the 'createTodo' function as it doesn't return any values.